src/test/java/StringCalculatorTest.java

```diff
@@ -0,0 +1,79 @@
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullSource;
+import org.junit.jupiter.params.provider.ValueSource;
+import static org.assertj.core.api.Assertions.*;
+
+
+class StringCalculatorTest {
+
+    private StringCalculator calculator;
+
+    @BeforeEach
+    void setUp() {
+        calculator = new StringCalculator();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"", "  "})
+    @NullSource
+    @DisplayName("빈 공백 문자일 경우, IllegalArgumentException 테스트")
+    void isEmpty_EnterCommand(String input) {
+        assertThatIllegalArgumentException().isThrownBy(
+                () -> {
+                    calculator.enterCommand(input);
+                }
+
+        ).withMessage("Input IllegalArgument Exception");
```

Comment:
예외처리 테스트 코드 작성 잘하셨습니다.✔
저도 처음에 이런 방식으로 코드를 작성했는데요
아래처럼 작성하는게 좀더 명확해보이고 일반적이라고 하네요. 참고해보세요😃
```suggestion
        assertThatThrownBy(() -> calculator.enterCommand(input))
                .isInstanceOf(IllegalArgumentException.class);
```